stripe.go

```diff
@@ -330,7 +330,7 @@ func (s *BackendImplementation) Do(req *http.Request, body *bytes.Buffer, v inte
 			// usually not used, but it doesn't hurt to set it in case it's
 			// needed. See:
 			//
-			//     https://github.com/stripe/stripe-go/issues/710
+			//     https://github.com/stripe/stripe-go/v55/issues/710
```

Comment:
Looks like an overly aggressive find + replace. Can you change this link and the one above it back to what it was?

Reply:
oops, sorry, that's what I get for throwing sed(1) at it and not inspecting the results closely enough. Thanks.